libs/sdk-ui-kit/src/ChartSorting/test/ChartSorting.test.tsx

```diff
@@ -140,10 +184,22 @@ describe("ChartSorting", () => {
     });
 
     describe("onApply", () => {
+        it("should not call onApply when disabled Apply button is clicked", () => {
+            const onApply = jest.fn();
+            const component = renderComponent({ onApply });
+            clickApplyButton(component);
+
```

Comment:
if it is easy what about to check disabled state/class of button too?

Reply:
Fixed.